frontend/src/pages/acceleratorProfiles/screens/manage/ManageAcceleratorProfile.tsx

```diff
@@ -19,12 +21,16 @@ type ManageAcceleratorProfileProps = {
 const ManageAcceleratorProfile: React.FC<ManageAcceleratorProfileProps> = ({
   existingAcceleratorProfile,
 }) => {
-  const [state, setState] = useGenericObjectState<AcceleratorProfileKind['spec']>({
+  const [state, setState] = useGenericObjectState<
+    { name?: string } & AcceleratorProfileKind['spec']
```

Comment:
You repeat this type multiple times... can we just make it a form type or something for the page? `AcceleratorProfileFormData` or something?